docs-chef-io/content/inspec/matchers.md

```diff
@@ -58,13 +79,13 @@ describe passwd.uid(0) do
 end
 ```
 
-`cmp` behaves in the following way:
+`cmp` matcher is allowed in the following comparisions:
 
-- Compare strings to numbers
+### Compares strings to numbers
```

Comment:
These probably shouldn't be headings (same with the other heading changes below). It sets off the text after the heading as being separate from the introduction sentence above.